diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter1/IdentifierNaming.kt

```diff
@@ -189,38 +198,55 @@ class IdentifierNaming(configRules: List<RulesConfig>) : DiktatRule(
                         }
                     }
                 } else if (variableName.text != "_" && !variableName.text.isLowerCamelCase()) {
+                    var isCorrectBackingField = false
+                    if (variableName.text.commonPrefixWith("_").length == 1 && variableName.text.drop(1).isLowerCamelCase()) {
```

Comment:
```suggestion
                    val  isCorrectBackingField  = if (variableName.text.commonPrefixWith("_").length == 1 && variableName.text.drop(1).isLowerCamelCase()) {
```
and instead of `matchingNode.let { isCorrectBackingField = true }`
`return matchingNode.let { true } ?: false`

Reply:
Done